src/components/DeveloperCard.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="card hoverable">
```

Comment:
You should wrap the developer card with `<div class="col s12 m4">` and its parent should have `<div class="row">` to be able to show the developer card like this:
![image](https://user-images.githubusercontent.com/5460365/81509471-ef15dc80-92d8-11ea-873b-584a6df414d9.png)

Reply:
I add a `col s12 m6 l4` m4 looked to cramped for ipad/tablets.